src/manifest.json

```diff
@@ -38,6 +36,7 @@
     "activeTab",
     "nativeMessaging",
     "notifications",
+    "scripting",
```

Comment:
Should this be in here? There is a separate definition for __chrome below and FF shouldn't need that with the current implementation, right?

Reply:
resolved